src/lando/ui/jinja2/stack/pull_request.html

```diff
@@ -0,0 +1,40 @@
+{% extends "partials/layout.html" %}
+{% block page_title %}{{pull_request}}{% endblock %}
+
+{% block main %}
+<main class="StackPage container fullhd">
```

Comment:
For my education: do we have some reference about the purposes of those classes?